dandi/metadata.py

```diff
@@ -302,6 +318,216 @@ def extract_field(field, metadata):
         return metadata.get(field, ...)
 
 
+nd_list = {
```

Comment:
also `nd_list` is a bad name for a dictionary ;)  so the file might get named `neurodata_types.json` or alike, and here `neurodata_types`